db/db_impl.cc

```diff
@@ -1153,6 +1154,28 @@ Status DBImpl::RecoverLogFiles(const std::vector<uint64_t>& log_numbers,
       }
       WriteBatchInternal::SetContents(&batch, record);
 
+#ifndef ROCKSDB_LITE
+      if (db_options_.wal_filter != nullptr) {
+        WALFilter::WALProcessingOption walProcessingOption =
```

Comment:
For acronyms, all caps also seems fine to me, although I can change to whichever you prefer.